DataConnectors/Gitlab/azuredeploy.json

```
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+      "PlaybookName": {
+            "defaultValue": "Gitlab-Sentinel",
```

Same comment as in README.md... should this be stylized to match their naming?

```suggestion
            "defaultValue": "GitLab-Sentinel",
```